content/sensu-go/5.14/guides/clustering.md

```diff
@@ -183,6 +183,50 @@ sensuctl cluster member-remove 2f7ae42c315f8c2d
 Removed member 2f7ae42c315f8c2d from cluster
 {{< /highlight >}}
 
+### Replace a faulty cluster member
+
+Here's how to replace a faulty cluster member to restore a cluster's health.
+
+First, run `sensuctl cluster health` to identify the faulty cluster member.
+For a faulty cluster member, the `Error` column will include an error message and the `Healthy` column will list `false`.
+
+In this example, cluster member `backend-4` is faulty:
+
+{{< highlight shell >}}
+sensuctl cluster health
+
+       ID            Name                          Error                           Healthy  
+────────────────── ─────────── ─────────────────────────────────────────────────── ─────────
+a32e8f613b529ad4   backend-1                                                        true
+c3d9f4b8d0dd1ac9   backend-2                                                        true
+c8f63ae435a5e6bf   backend-3                                                        true
+2f7ae42c315f8c2d   backend-4  dial tcp 10.0.0.4:2379: connect: connection refused   false
+
+{{< /highlight >}}
+
+Second, delete the faulty cluster member. To continue this example, you will delete cluster member `backend-4` using its ID field:
+
+{{< highlight shell >}}
+sensuctl cluster member-remove 2f7ae42c315f8c2d
+
+Removed member 2f7ae42c315f8c2d from cluster
+{{< /highlight >}}
+
+Third, add the faulty cluster member back to the cluster. Make sure to change the `etcd-initial-cluster-state` to `existing` in the configuration.
```

Comment:
```suggestion
Third, add a newly created member to the cluster. You can use the same name and IP address as the faulty member you deleted, with one change to the configuration: specify the `etcd-initial-cluster-state` as `existing`.
```

Reply:
@rgeniesse what do you think of this revision?

Reply:
Looks great @hillaryfraley